### PR TITLE
Lavaland closed for renovations

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I would like some time to process all the feedback I've gotten, work on it, and return lavaland better than it is now. 

EDIT: To be clear this is not a long term thing. I'll be on vacation until sunday, then hopefully have a roadmap or something in a few days for what I want to change after that, and then have it back up within a couple weeks for more testing. It will make everything saner going forward if I can take a short breather and tell people what goals I want fulfilled instead of people (including me) taking random stabs in the dark at helping.

:cl: Kor
rscadd: Lavaland is closed for renovations.
/:cl:

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Godspeed

**Comment by coiax** [comment]
BOOO, ASTEROID IS SHIT

File: _maps/dreamstation.dm [hunk 1]
```diff
@@ -17,13 +17,13 @@ z7 = empty space
...
 
-		#define MINETYPE "lavaland"
+		#define MINETYPE "mining"
 
 		#include "map_files\DreamStation\dreamstation04.dmm"
 		#include "map_files\generic\z2.dmm"
 		#include "map_files\generic\z3.dmm"
 		#include "map_files\generic\z4.dmm"
-		#include "map_files\generic\lavaland.dmm"
+		#include "map_files\generic\z5.dmm"
```
Comment (ChangelingRain) [inline _maps/dreamstation.dm]: #returndreamstationz5

**Comment by KazeEspada** [comment]
THE ACTUAL END OF LAVALAND! I feel bad for that joke now.

**Comment by drynwyn** [comment]
...Why is it it can't stay in the game in the mean time? Lavaland is way, way better than the asteroid.

**Comment by Cheridan** [comment]
agree with @drynwyn

**Comment by KorPhaeron** [comment]
Yeah but I'm tired and about to go on a short vacation, and I don't want players to burn out (get it?) on lavaland while I take a break from working on it.

**Comment by Durkel** [comment]
The burn out is real.

**Comment by lordpidey** [comment]
@KorPhaeron Would you like me to help you by adding some stuff from the demon gamemode?

**Comment by optimumtact** [comment]
This is dumb, why are you wasting your own work?

**Comment by Durkel** [comment]
I think it's to prevent a situation like cult/ling from occurring, With too many people sticking their dicks into the mix and it turning into a unrecognizable clusterfuck. Probably want's to preserve his ideal image of lavaland while he's gone.

**Comment by optimumtact** [comment]
When you gift things to the codebase you lose a lot of control of them, if you're not ready to make that call then you shouldn't really be adding things

**Comment by drynwyn** [comment]
Kor is literally in charge of keeping too many people from sticking their dicks into things and making a clusterfuck of them, though.

**Comment by Incoming5643** [comment]
People don't have exclusive control of their features once they pass the pull request phase. You can't remove something people like for personal reasons. :-1:

**Comment by Fox-McCloud** [comment]
I can understand the need/want to do this, and or that, I can respect the opinion and support it. As it stands now, Lavaland is...interesting, but IMO has a large number of problems. I say :+1: to this so that we can get a better iteration of lavaland, later.

**Comment by Zilenan91** [comment]
Gonna go with a big :-1: 

It's much better than the asteroid is currently. Just tell people that you'll be on vacation and not to leave feedback on it until you're back.

**Comment by KorPhaeron** [comment]
It has problems that need fixing though

**Comment by KazeEspada** [comment]
It better have a boss fight next time it comes around, you hear?

**Comment by Zilenan91** [comment]
Yeah but the asteroid has way more problems than lavaland does, it's going from pretty okay to not okay on the scale.

**Comment by duncathan** [comment]
kn-kn-kn-kn-KNEEJERK

**Comment by duncathan** [comment]
;)

**Comment by KorPhaeron** [comment]
libera me from lavaland

**Comment by Thunder12345** [comment]
Even Travis wants this closed

**Pull request closed without merging**